src/libs/protocols/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "protocols"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[features]
+default = []
+with-serde = [ "serde", "serde_json" ]
+
+[dependencies]
+#ttrpc = { version = "0.5.0", features = ["async"] }
```

Comment:
Commented out code.

Reply:
related.. line below is pulling from a fork. Is there something required in there that isn't in upstream ttrpc?

Reply:
> related.. line below is pulling from a fork. Is there something required in there that isn't in upstream ttrpc?

hi dan, current occlum can't support UDS. Offical ttrpc can't support tcp, so we add tcp support in ttrpc. Once the Occlum NGO can support UDS, we will use the upstream ttrpc

Reply:
> Commented out code.

Get it. Done